test/BUILD

```diff
@@ -1,10 +1,11 @@
 cc_test(
     name = "tests",
-    srcs = glob(["**/*.cc"]),
```

Comment:
I would personally avoid using such generic regex expressions for build targets. This can lead to future files unintentionally being added to this target (Whenever someone adds a file in this directory, they would need to check if it doesn't affect this target, even if they didn't intend to change any existing modules).

With my above suggestion of having many build targets for tests, this should not be necessary.